### PR TITLE
[Image Foundation models] Minor Fixes

### PR DESCRIPTION
**Pull request by rjaincc**
# Description
* Fixing name casing
* Adding warning for ME compute in FT notebooks for OD and IS
* Updating model versions 

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [x] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [x] Pull request includes test coverage for the included changes.
- [x] This notebook or file is added to the [CODEOWNERS](https://github.com/Azure/azureml-examples/blob/main/.github/CODEOWNERS) file, pointing to the author or the author's team.

### CODE REVIEW

File: sdk/python/foundation-models/system/finetune/image-object-detection/mmdetection-fridgeobjects-object-detection.ipynb [hunk 1]
```diff
@@ -170,7 +170,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "finetune_cluster_name = \"sample-finetune-cluster-gpu\"\n",
+    "finetune_cluster_name = \"gpu-nc6sv3-ssh-enabled\"  # \"sample-finetune-cluster-gpu\"\n",
```
Comment (vivek-dani) [inline sdk/python/foundation-models/system/finetune/image-object-detection/mmdetection-fridgeobjects-object-detection.ipynb:173]: did we want "sample-finetune-cluster-gpu" ?

File: sdk/python/foundation-models/system/inference/image-classification/image-classification-batch-endpoint.ipynb [hunk 2]
```diff
@@ -239,7 +239,7 @@
     "\n",
     "We can provide input images to batch inference either in a folder containing images or in a csv file containing \"image\" named column having images in base64 format.\n",
     "\n",
-    "Note: If job failed with error Assertion Error (`The actual length exceeded max length 100 MB`) then please try with less number of input images or use ImageFolder Input mode."
+    "The deployment in the \"Create batch deployment\" section below takes the argument `mini_batch_size`, which is the number of CSV files processed by the model in a single mini_batch."
```
Comment (vivek-dani) [inline sdk/python/foundation-models/system/inference/image-classification/image-classification-batch-endpoint.ipynb:242]: is my understanding correct ?
mini_batch_size= number of rows in single csv file ?
number of mini batches == number of csv files ?

I am little confused here as different csv can have different number of rows.

File: sdk/python/foundation-models/system/inference/image-classification/image-classification-batch-endpoint.ipynb [hunk 3]
```diff
@@ -282,11 +282,11 @@
...
+    "Batch endpoints are endpoints that are used to do batch inferencing on large volumes of data over a period of time. The endpoints receive pointers to data and run jobs asynchronously to process the data in parallel on compute clusters. Batch endpoints store outputs to a data store for further analysis. For more information on batch endpoints and deployments see [What are batch endpoints?](https://learn.microsoft.com/en-us/azure/machine-learning/concept-endpoints?view=azureml-api-2#what-are-batch-endpoints) In this sub-section, we will cover the following items:\n",
     "\n",
     "* Create a batch endpoint.\n",
     "* Create a batch deployment.\n",
-    "* Set the deployment as default; doing so allows invoking the endpoint without specifying the deployment's name."
+    "* Set the deployment as default: doing so allows invoking the endpoint without specifying the deployment's name."
```
Comment (vivek-dani) [inline sdk/python/foundation-models/system/inference/image-classification/image-classification-batch-endpoint.ipynb:289]: nit: 
as default. Doing

File: sdk/python/foundation-models/system/finetune/image-object-detection/mmdetection-fridgeobjects-object-detection.ipynb [hunk 4]
```diff
@@ -203,23 +203,41 @@
...
     "    )\n",
-    "    workspace_ml_client.begin_create_or_update(compute_config).result()"
+    "    workspace_ml_client.begin_create_or_update(compute_config).result()\n",
+    "\n",
+    "model_evaluation_compute_instance_type = model_evaluation_compute.size\n",
+    "print(\n",
+    "    f\"Model Evaluation compute's instance type: {model_evaluation_compute_instance_type}\"\n",
+    ")\n",
+    "\n",
+    "if model_evaluation_compute_instance_type != \"STANDARD_NC6S_V3\":\n",
```
Comment (vivek-dani) [inline sdk/python/foundation-models/system/finetune/image-object-detection/mmdetection-fridgeobjects-object-detection.ipynb:232]: should we allow if its bigger than STANDARD_NC6S_V3 ?

**Review by vivek-dani — APPROVED** [review]

**Review by nvijayrania — APPROVED** [review]

**Pull request merged**